h5netcdf/tests/test_h5netcdf.py

```diff
@@ -1774,10 +1774,10 @@ def test_track_order_specification(tmp_local_netcdf):
 # This should always work with the default file opening settings
 # https://github.com/h5netcdf/h5netcdf/issues/136#issuecomment-1017457067
 def test_more_than_7_attr_creation(tmp_local_netcdf):
-    with h5netcdf.File(tmp_local_netcdf, "w") as h5file:
+    with h5netcdf.File(tmp_local_netcdf, "w") as _h5file:
```

Comment:
The changes here look unintentional?

Reply:
:man_facepalming: OMG, thanks for spotting this. I've stepped through the search and replace one by one, but it looks I was too confident that everything will be OK.

Reply:
Unfortunately this is already released as h5netcf 1.6.4. I'll fix this on main now. Do you see immediate need of following up with another release or can we postpone to some later date?